tests/tracer/test_context.py

```diff
@@ -88,6 +89,24 @@ def validate_traceparent(context, sampled_expected):
         Context(trace_id=123, span_id=321),
         Context(trace_id=123, span_id=321, dd_origin="synthetics", sampling_priority=2),
         Context(trace_id=123, span_id=321, meta={"meta": "value"}, metrics={"metric": 4.556}),
+        Context(
+            trace_id=123,
+            span_id=321,
+            meta={"meta": "value"},
+            metrics={"metric": 4.556},
+            span_links=[
+                SpanLink(
+                    trace_id=123 << 64,
+                    span_id=456,
+                    tracestate="congo=t61rcWkgMzE",
+                    flags=1,
+                    attributes={"link.name": "some_name"},
+                )
+            ],
+        ),
+        Context(
```

Comment:
Does this test fail without the functional change? When I tried adding a similar test case I couldn't get it to fail.

Reply:
No I think the test will always pass. I'll modify. 

You need to access the unserialized fields on the context object to raise the attribute error.

Reply:
@emmettbutler I found the issue: 6c7aa449227ee66d0c64564c2dc0df03b8cfdce9. The test was not checking the span_links attribute